webplugin/js/app/kommunicate-ui.js

```diff
@@ -491,6 +415,33 @@ KommunicateUI = {
             );
             MCK_BOT_MESSAGE_QUEUE = [];
             if (MCK_EVENT_HISTORY.length >= 2) {
+                if(
+                    MCK_EVENT_HISTORY[MCK_EVENT_HISTORY.length - 2] ==
+                    'km-faq-category-list'
+                ){
+                    KommunicateUI.showHeader();
+                    $applozic('#km-faqdiv')
```

Comment:
what is common between all these elements may be parent container, or something else? if not, then can we make something common such as adding a specific class
then replace this code with one just one of code that add and remove class based on that common identifer.